app/addons/setup/setup.js

```diff
@@ -179,6 +238,10 @@ var SetupMultipleNodesController = React.createClass({
     SetupActions.setPortForSetupNode(e.target.value);
   },
 
+  alterNodeCount: function (e) {
+    SetupActions.setNodeCount(e.target.value);
```

Comment:
I was just copying how all of the other functions were doing it. If I change this one, do I have to change all the others? Really out of my depth here